.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
```

Comment:
This workflow is not being executed: https://github.com/autogluon/autogluon-bench/actions/runs/4506390243

I'm not sure what does that error msg mean, and `checkout` and `setup-python` should be official action supported by github. You might want to research it a bit